### ISSUE TITLE
Auto-assign host on single-host queue

### ISSUE THREAD

**Issue by mfldavidson** [tl-its-umich-edu/remote-office-hours-queue#227]
If a queue has only one host, auto-assign all meetings in the queue to the only host

**Comment by mfldavidson** [comment]
This is a nice to have, but it is on the lower-priority end--don't start until all other 1.4 issues have been addressed, time permitting
